### PR TITLE
llzip

### PR DESCRIPTION
**Pull request by MarikoAlvarado**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,4 @@
 
 - [Linked List Insertions](javascript/code-challenges/linked-list/ll-insertions/README.md)
 
+- [Linked List Zip](javascript/code-challenges/llZip/README.md)
```
Comment (adrianhuebner) [inline README.md:15]: What a wonderful whiteboard image! 
I would love to see some code and tests added to this!
